{"name": "test_get_users_api", "status": "failed", "statusDetails": {"message": "AssertionError: API call failed, expected status code 200.\nassert 401 == 200\n +  where 401 = <Response [401]>.status_code", "trace": "self = <tests.test_api.TestAPITesting object at 0x0000020E1E300E60>\n\n    @pytest.fixture(scope=\"class\")\n    def get_users_response(self):\n        \"\"\"Fixture to get the API response for user details.\"\"\"\n        url = \"https://opensource-demo.orangehrmlive.com/web/index.php/api/v2/admin/users?limit=50&offset=0&sortField=u.userName&sortOrder=ASC\"\n        response = requests.get(url)\n>       assert response.status_code == 200, \"API call failed, expected status code 200.\"\nE       AssertionError: API call failed, expected status code 200.\nE       assert 401 == 200\nE        +  where 401 = <Response [401]>.status_code\n\n..\\trial\\tests\\test_api.py:37: AssertionError"}, "start": 1728568945376, "stop": 1728568945376, "uuid": "c87b041b-9ece-46e1-9736-8f8747c55315", "historyId": "7b0fd47b769697208234c378717b9876", "testCaseId": "7b0fd47b769697208234c378717b9876", "fullName": "tests.test_api.TestAPITesting#test_get_users_api", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestAPITesting"}, {"name": "host", "value": "Vaishnavi"}, {"name": "thread", "value": "3464-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}]}